### DISCUSSION TITLE
gist.github.com cannot input text in Chrome

### DISCUSSION THREAD

**Discussion by sum2012** [community/community#53634] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

In https://gist.github.com create new gist in windows 7 Chrome v109.0.5414.120 ,I cannot enter text because cannot get focus.I think that it worked from last  week .Now I only can use Opera.

**Comment by mrepol742** [comment]
Have you tried updating Google Chrome? as of this moment the latest version is 112.0.5615.121 and also have you try resetting the preference/settings?

**Reply by sum2012** [reply, marked as answer]
I find that is is Tampermonkey plugin problem
